src/main/java/wolox/training/model/Book.java

```diff
@@ -0,0 +1,125 @@
+package wolox.training.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    private String genre;
+
+    @Column(nullable = false)
```

Comment:
Just so you know, you can use the @NotNull annotation from jpa for this cause, but it's not wrong as you did it.

Reply:
I think [@NotNull annotation](https://docs.oracle.com/javaee/7/api/javax/validation/constraints/NotNull.html) is used to specify that element can't be Null in Java. But the [@Column(nullable = false)](https://docs.oracle.com/javaee/5/api/javax/persistence/Column.html) avoids to persist NULL in that column
I thought that the columns shouldn't allow NULLs because there is another card on which the setters check NULLs

Reply:
You might be right, but I remember using the @NotNull for both things. Check it out ;) Otherwise, it's great as you have it.

Reply:
I just tried it and it works! You are totally right! :smile: